airbyte-integrations/bases/base-normalization/integration_tests/resources/test_primary_key_streams/dbt_schema_tests/schema_test.yml

```diff
@@ -4,11 +4,11 @@ models:
   - name: exchange_rate
     tests:
       - dbt_utils.equality:
-          description: check_streams_are_equal
-            In this integration test, we are sending the same records to both streams
-            exchange_rate and dedup_exchange_rate.
-            The SCD table of dedup_exchange_rate in append_dedup mode should therefore mirror
-            the final table with append or overwrite mode from exchange_rate.
+          # description: check_streams_are_equal
+          #  In this integration test, we are sending the same records to both streams
+          #  exchange_rate and dedup_exchange_rate.
+          #  The SCD table of dedup_exchange_rate in append_dedup mode should therefore mirror
+          #  the final table with append or overwrite mode from exchange_rate.
```

Comment:
If I don't comment the `description` out, DBT will throw the following error:

```
macro 'dbt_macro__mysql__test_equality' takes no keyword argument 'description'
```
which makes sense, because the `description` is not a parameter for this macro. But I am confused why this error is not throw for other DB or tests.

Reply:
Weird that it's breaking only now...

 I've been putting these descriptions fields for documenting our tests. Commenting them out are fine too though...